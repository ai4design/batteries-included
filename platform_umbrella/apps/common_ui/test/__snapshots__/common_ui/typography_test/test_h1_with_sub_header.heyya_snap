<h1 class="text-3xl text-pink-500 sm:text-5xl font-extrabold my-6">
  
  Header<span class="m-0 text-gray-700">::<span class="font-mono">with sub</span></span>
</h1>