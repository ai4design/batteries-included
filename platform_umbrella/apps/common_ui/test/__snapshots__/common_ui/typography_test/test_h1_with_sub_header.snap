"<h1 class=\"text-3xl text-pink-500 sm:text-5xl font-extrabold my-6\">\n  \n  Header\n  <span class=\"m-0 text-gray-700\">::</span>\n  <span class=\"text-gray-700 font-mono\">\n    with sub\n  </span>\n</h1>"